examples/12_rerank.yaml

```diff
@@ -0,0 +1,75 @@
+# Using a trained model to do reranking score for an nbest list
+
+exp1-model: !Experiment
+  exp_global: !ExpGlobal
+    # The model file contain the whole contents of this experiment in YAML
+    # format. Note that {EXP} expressions are left intact when saving.
+    model_file: examples/output/{EXP}.mod
+    log_file: examples/output/{EXP}.log
+    default_layer_dim: 64
+    dropout: 0.5
+    weight_noise: 0.1
```

Comment:
Don't you need a `pretrained_model_file` here if we're loading a pretrained model? Perhaps it'd be good to have exp1 train a model, then exp2 actually perform the reranking.